apps/namadillo/README.md

```diff
@@ -38,31 +51,40 @@ yarn lint:fix
 yarn test
 ```
 
-[ [Table of Contents](#table-of-contents) ]
-
 ### Configuration
 
-Configuration is done by creating a `.env` file, based on [.env.sample](./.env.sample), and specifying the values you wish to override.
+You can configure the application by modifying the `config.toml` file, located in the public folder. Use the [template provided](./public/config.toml) as a base, and specify the values you wish to override.
```

Comment:
@pedrorezende do you think it worth mentioning that you can setup this directly on the interface and the toml file is an alternative method?